spring-cloud-huawei-service-engine/service-engine-common/src/main/java/com/huaweicloud/service/engine/common/configration/bootstrap/DiscoveryBootstrapProperties.java

```diff
@@ -32,7 +32,7 @@ public class DiscoveryBootstrapProperties {
 
   private boolean watch = false;
 
-  private boolean enableZoneAware = false;
+  private boolean enabledZoneAware = false;
```

Comment:
Why change this?

Reply:
Consistent with the fliter condition

@ConditionalOnProperty(value = "spring.cloud.servicecomb.discovery.enabledZoneAware", havingValue = "true")